e2e/auth.cleanup.ts

```diff
@@ -0,0 +1,58 @@
+import { Page, test as cleanup, expect } from '@playwright/test'
+import { ADMIN_STORAGE_STATE, USER_STORAGE_STATE } from '../playwright.config'
+import { getEndpoint } from './helpers/http'
+import {
+  deleteTestUser,
+  getAccessTokenFromCookie,
+  getUserIdFromLocalStorage,
+  goToDashboard,
+  logoutTestUser,
+} from './helpers/testUser'
+
+let adminPage: Page
+let userPage: Page
+
+cleanup.describe('Clean up', () => {
+  cleanup.beforeAll(async ({ browser }) => {
+    adminPage = await browser.newPage({
+      storageState: ADMIN_STORAGE_STATE,
+    })
+    userPage = await browser.newPage({
+      storageState: USER_STORAGE_STATE,
+    })
+  })
```

Comment:
Create new pages in `beforeAll`, so they can be closed in `afterAll`, even if a step within the test fails. Create the pages using the stored state, so the user and admin are logged in on their respective pages.